.github/workflows/pr_notification.yml

```
@@ -0,0 +1,140 @@
+                throw new Error(`Discord ID not found for GitHub user: ${existingReviewer}`);
+              }
+
+              core.setOutput('reviewer', existingReviewer);
+              core.setOutput('discord_id', discordId);
+            }
+
+      - name: Send Discord Notification
+        env:
+          DISCORD_WEBHOOK_URL: ${{ secrets.DISCORD_WEBHOOK_URL }}
+
+        run: |
```

actionlint (1.7.7)

"github.event.pull_request.title" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/security-for-github-actions/security-guides/security-hardening-for-github-actions for more details

(expression)